migration/migrator/migrations/course/20190415092504_null_email_address.py

```diff
@@ -0,0 +1,7 @@
+def up(config, database, semester, course):
+    database.execute("ALTER TABLE only users ALTER COLUMN user_email DROP NOT NULL")
+    pass
+
+
+def down(config, database, semester, course):
```

Comment:
Please add a down function for adding this migrator script to add the constraint back. If this change ever needs to be rolled back on it will make things easier.